packages/react-instantsearch/src/__tests__/common-widgets.test.tsx

```diff
@@ -242,6 +242,32 @@ const testSetups: TestSetupsMap<TestSuites> = {
       </InstantSearch>
     );
   },
+  createCurrentRefinementsWidgetTests({
+    instantSearchOptions,
+    widgetParams,
+    testParams,
+  }) {
+    const refinementListAttributes = Object.keys(
+      instantSearchOptions.initialUiState?.indexName?.refinementList || {}
+    );
+    const search = (
+      <InstantSearch {...instantSearchOptions}>
+        {refinementListAttributes.map((attribute) => (
+          <RefinementList key={attribute} attribute={attribute} />
```

Comment:
we need to test all widget and refinement types

Reply:
I'll close this comment thread and we can continue the conversation at https://github.com/algolia/instantsearch/pull/5838#discussion_r1319655476.